src/main/java/fi/helsinki/cs/tmc/cli/command/LoginCommand.java

```diff
@@ -51,7 +49,15 @@ public void run(CliContext context, CommandLine args) {
             return;
         }
 
-        if (!ctx.loadBackendWithoutLogin()) {
+        if (this.ctx.checkIsLoggedIn(true, true)) {
+            Optional<String> username = this.ctx.getSettings().getUsername();
```

Comment:
getOrDefault?
same below

Reply:
I don't see how getOrDefault would help here since we want to concatinate stuff to the vale of the optional